### DISCUSSION TITLE
任务栏高度低于5%时日期和时间才单行显示是否合理

### DISCUSSION THREAD

**Discussion by ankium** [linuxdeepin/developer-center#4482] [General | 常规讨论]
![avatar](https://storage.deepin.org/thread/202305232055103915_%E5%BE%AE%E4%BF%A1%E6%88%AA%E5%9B%BE_20230523203859.png)
如图，V23 Beta版本中当设置任务栏大小（实际就是高度）约在5%的尺寸时，任务栏的日期和时间才会单行显示，而在大于5%时，日期和时间是上下并排显示；通过在不同分辨率的屏幕对比，发现5%这个临界值太小，希望官方能够优化这个选项，或者让用户自行决定在任务栏多高时日期和时间单行/并排显示！现阶段不少社区用户都表示日期和时间单行显示效果更美观！

**Comment by BLumia** [comment]
说实话我觉得这部分整个交互都很诡异，日期与时间的格式、日期时间布局变化、点击日期与时间时的行为都不太对劲。

后续 dock 会重新设计的更灵活一些，允许用户整个自定义 dock 的布局，到时候应该无论设计给出的方案是什么，用户都可以自己自定义一份符合自己需求与行为预期的 dock 栏。

**Reply by ankium** [reply]
V23版本的dock会重新设计？

**Reply by BLumia** [reply]
虽然目前就已经是重新设计过的了，但有重构的计划。默认提供的样式与交互应该和 beta 里的基本一样，但会预留更多的定制空间给用户（以及潜在的 OEM 行为）

**Reply by ankium** [reply]
期待
